schemas/v1.0/readme.md

```diff
@@ -0,0 +1,36 @@
+# OpenAPI Overlay 1.0.x JSON Schema
+
+Here you can find the JSON Schema for validating OpenAPI Overlay description of versions 1.0.x.
```

Comment:
```suggestion
Here you can find the JSON Schema for validating Overlays of versions 1.0.x.
```
See [Style guide for Overlay Specification](https://github.com/OAI/Overlay-Specification/blob/main/CONTRIBUTING.md#style-guide-for-overlay-specification)